eras/shelley/impl/src/Cardano/Ledger/Shelley/BlockChain.hs

```diff
@@ -250,27 +250,33 @@ bbHash (TxSeq' _ bodies wits md) =
     hashStrict = Hash.hashWith id
     hashPart = Hash.hashToBytes . hashStrict . BSL.toStrict
 
-data Block era
-  = Block' !(TP.BHeader (Crypto era)) !(Era.TxSeq era) BSL.ByteString
+data Block h era
```

Comment:
Now that `Block` is parametric over the header type, I wonder whether we should move it to `cardano-ledger-core`